.pre-commit-config.yaml

```diff
@@ -44,6 +44,14 @@ repos:
     hooks:
       - id: flake8
 
+  - repo: https://github.com/Riverside-Healthcare/djLint
+    rev: v1.31.0
+    hooks:
+      - id: djlint-reformat-jinja
+        types_or: [html]
+      - id: djlint-jinja
+        types_or: [html]
```

Comment:
That's nice that we can lint/reformat templated files, but is that actually working? Asking as we exclude the whole templated project on the first line of this config:

https://github.com/cookiecutter/cookiecutter-django/blob/7ad2e1fdae82e011fa823f5f42590bf6ce3638ee/.pre-commit-config.yaml#L1

Reply:
Yes, thank you, I hadn't noticed. I ended up removing the pre-commit hook after all.